### PR TITLE
Joe security bug

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25221

## Description
When running "joe-analysis-info" on URl, never returns a DBotScore.Indicator

## Screenshots
image

### CODE REVIEW

File: Packs/JoeSecurity/pack_metadata.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-    "Forensics & Malware Analysis"
-  ],
-  "tags": [],
-  "useCases": [],
-  "keywords": []
-}
+    "name": "Joe Security",
+    "description": "Sandbox Cloud",
+    "support": "xsoar",
+    "currentVersion": "1.1.0",
```
Comment (ShahafBenYakir) [inline Packs/JoeSecurity/pack_metadata.json:5]:
```suggestion
    "currentVersion": "1.0.1",
```

File: Packs/JoeSecurity/ReleaseNotes/1_1_0.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (ShahafBenYakir) [inline Packs/JoeSecurity/ReleaseNotes/1_1_0.md:1]: Change file name to 1_0_1.md

File: Packs/JoeSecurity/Integrations/JoeSecurity/JoeSecurity.py [hunk 3]
```diff
@@ -102,7 +102,7 @@ def analysis_to_entry(title, info):
 
         dbot_scores.append({
             'Vendor': 'JoeSecurity',
-            'Indicator': analysis_info['MD5'],
+            'Indicator': analysis_info['MD5'] if analysis_info['MD5'] else analysis['filename'],
```
Comment (ShahafBenYakir) [inline Packs/JoeSecurity/Integrations/JoeSecurity/JoeSecurity.py:105]: The bug was that `analysis_info` did not have an `MD5` key? if this is the case we will get an exception here too.

**Comment by ShahafBenYakir** [comment]
Also, can you run the new Test PB on a demisto that doesn't have the fix in the integration and attach a screenshot so we could see the failure?

File: Packs/JoeSecurity/ReleaseNotes/1_0_2.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+- __Joe Security__
+Fixed a bug that running "joe-analysis-info" on URL not returns a DBotScore.Indicator.
```
Comment (ShahafBenYakir) [inline Packs/JoeSecurity/ReleaseNotes/1_0_2.md:4]:
```suggestion
Fixed a bug in **joe-analysis-info** command that wasn't returning a DBotScore.Indicator when was given a URL.
```

File: Packs/JoeSecurity/Integrations/JoeSecurity/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+- Fixed a bug that running "joe-analysis-info" on URL not returns a DBotScore.Indicator.
```
Comment (ShahafBenYakir) [inline Packs/JoeSecurity/Integrations/JoeSecurity/CHANGELOG.md:2]:
```suggestion
- Fixed a bug in **joe-analysis-info** command that wasn't returning a DBotScore.Indicator when was given a URL.
```

**Comment by Itay4** [comment]
build fails on marketplace 
merging

**Pull request merged**